Source/RapyutaSimulationPlugins/Private/Robots/Turtlebot3/TurtlebotBurger.cpp

```diff
@@ -55,6 +59,17 @@ void ATurtlebotBurger::SetupBody()
     SetupConstraintsAndPhysics();
 }
 
+void ATurtlebotBurger::PreInitializeComponents()
+{
+    Super::PreInitializeComponents();
+    if (IsDynamicRuntimeRobot() && DefaultRoot->GetAttachParent())
+    {
+        // NOTE: This requires [DefaultRoot] to have been attached to some NEW Root, by which there is no need to promote its child

```

Comment:
when "[DefaultRoot] to have been attached to some NEW Root" happen?

Reply:
That only happens if the RootComp has been switched to a new one earlier, and thus parenting DefaultRoot. But this part is actually unrequired here due to TurtlebotBurger is statically-built class, so removed.